website/blog/2022-12-29-sqllogictest-framework.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Rewrite sqllogictest framework in rust
+description: sqllogictest, rust
+date: 2022-12-29
+tags: [databend, sqllogictest, rust]
+authors:
+- name: Xudong
+  url: https://github.com/xudong963
+  image_url: https://github.com/xudong963.png
+---
+
+Sqllogictest is a program designed to verify that an SQL database engine computes correct results by comparing the results to identical queries from other SQL database engines. Sqllogictest was originally designed to test [SQLite](http://www.sqlite.org/), but it is database engine neutral and can just as easily be used to test other database products.
```

Comment:
We can link sqllogictest to https://www.sqlite.org/sqllogictest/doc/trunk/about.wiki